.husky/pre-commit

```diff
@@ -1,4 +1,9 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-yarn pre-commit
+if [ -n "$(git diff --name-only --staged | grep 'yarn.lock')" ]; then
+  yarn dedupe
+  git add -u
+fi
+
+yarn lint-staged
```

Comment:
How come we can run a script that does not exist? And will removing the `pre-commit` script not remove formatting? 😕

Reply:
wdym? `yarn lint-staged` is provided by the `lint-staged` package.. and previously, we're redirecting git pre-commit to `yarn pre-commit`.. which doesn't make sense.. why should `git` depends on a script being defined in `package.json` instead of calling the intended command directly??

Reply:
Ah, for some reason I thought you removed the dep and related script together when I wrote this. All good